core/types/transaction_conditional.go

```diff
@@ -0,0 +1,256 @@
+package types
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"math/big"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/rlp"
+)
+
+// KnownAccounts represents a set of KnownAccounts
+type KnownAccounts map[common.Address]KnownAccount
+
+// EncodeRLP will encode the known account into rlp bytes
+func (ka KnownAccounts) EncodeRLP(w io.Writer) error {
```

Comment:
Why do we need RLP encoding of the known accounts?

Reply:
RLP encoding is here so that the metadata is gossiped along with the tx

Reply:
might end up removing this depending on discussion